packages/obonode/obojobo-chunks-math-equation/math-equation-properties-modal.scss

```diff
@@ -32,3 +36,18 @@
 		}
 	}
 }
+
+#katexContainer {
```

Comment:
* Always prefer to use classes instead of ids in any component.
* This style should be nested inside the `.math-equation-properties` rule. As is this rule is now going to globally style anything on the page with an id of 'katexContainer'. While it's unlikely you still can't be 100% certain that another developer created a content type in Obojobo that is also using an element with an id of `katexContainer`.